torch/_inductor/codecache.py

```diff
@@ -167,6 +172,7 @@ def get_system() -> Dict[str, Any]:
         return system
 
     @staticmethod
+    @clear_on_fresh_inductor_cache
```

Comment:
I did not try to modify the behavior of `get_global_cache_path` below since it seems like a different thing altogether, but lemme know if this is something that should also reset with `fresh_inductor_cache`